src/components/Info/styles.styl

```diff
@@ -0,0 +1,24 @@
+@require 'settings/palette.styl'
+@require 'tools/mixins'
+
+.infos
+    background paleGrey
+    display flex
+    align-items center
+    background-color #fff2f2
```

Comment:
Do not use color code anymore, only [variable](https://docs.cozy.io/cozy-ui/styleguide/section-settings.html) or rgba black or white.

Same on https://github.com/cozy/cozy-banks/pull/697/files#diff-2b34118a76524e9121686aa071cf6a28R24